nyc_taxi/nyc_taxi-nongeo.ipynb

```diff
@@ -134,18 +128,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "scatter = hv.Scatter(df, 'trip_distance', 'fare_amount')\n",
-    "ranged = scatter.redim.range(trip_distance=(0, 20), fare_amount=(0,40))\n",
-    "shaded = hd.spread(hd.datashade(ranged))\n",
-    "labelled = shaded.redim.label(trip_distance=\"Distance, miles\", fare_amount=\"Fare, $\")\n",
-    "labelled"
+    "df.hvplot.scatter('trip_distance', 'fare_amount', rasterize=True, dynspread=True, cnorm='eq_hist',\n",
+    "                  xlabel='Distance, miles', ylabel='Fare, $', xlim=(0,15), ylim=(0,40), alpha=0.05)"
```

Comment:
The plot looks quite different with these changes, the points on the new plot are quite small, they're a bit hard to see in the bottom part of the plot.

Before:
<img width="821" alt="image" src="https://github.com/holoviz-topics/examples/assets/35924738/fd3d2fc6-d55b-41cd-869d-44e3f3f92851">

Now:
<img width="817" alt="image" src="https://github.com/holoviz-topics/examples/assets/35924738/f0001370-0dfb-4019-bdd7-13169724debe">

I played around a little with it, it seems like it's mostly due to using `dynspread` instead of `spread`. The `spread` operation is not exposed to hvPlot users, there's I assume a good reason for that. Let's ask Jim's opinion.

Reply:
I tried removing the `alpha` parameters and it became better, although still not as visible as the old one.

Reply:
Ok, let's try to discuss this during our meeting tomorrow.

Reply:
@Azaya89 can you please add these parameters to hvPlot calls that set `dynspread=True`? `threshold=1, max_px=1`.

Also, please make sure to arrange the parameters in the following order (when applicable): `rasterize`, `cnorm`, `dynspread`, `threshold`, `max_px`. Basically, let's put together parameters that are related. This applies to other parameters like xlim/ylim or xlabel/ylabel, I think we already do a good job for others but make sure it's the case.

<img width="823" alt="image" src="https://github.com/holoviz-topics/examples/assets/35924738/0661cad8-c867-4b4a-9943-d65c8ec0f036">